### ISSUE TITLE
NGMPP material in the backlog

### ISSUE THREAD

**Issue by eroux** [buda-base/library-issues#328]
Hi @karmagongde ,

We've discussed with Jann and it appears it would be helpful to prioritize the NGMPP material in the backlog. Could you catalog it and put in ReadyToProcess ? I've identified NGMPP material in the following directories:
- /Incoming/scans/Inbox/microfilm and microfiche/microfilm
- /Incoming/scans/Inbox/Sheehy Hard Drive Recovered/Hirshberg NGMP pdfs

**Comment by TBRC-Travis** [comment]
@karmagongde let me know once the cataloging is complete and I'll help you get this material moved over to ReadyToProcess

**Comment by karmagongde** [comment]
@TBRC-Travis @eroux  These are the part of backlog and librarians have already cataloged in 2017. All the work numbers and IG numbers are on the Google spreadsheet below. There are three tabs. 1. RS3: Photocopies from Hamburg; 2. RS3: Sheehy Hard Drive Recovered; 3. RS4: Sheehy. 
https://docs.google.com/spreadsheets/d/10BcOQeMkcPO79YvEgODLKsiZ412bAQs-Gi3HfZIXFSE/edit#gid=2043111283

**Comment by TBRC-Travis** [comment]
@karmagongde ok, great.  I'll take care of these.

**Comment by eroux** [comment]
thanks both! @karmagongde are you sure the files in `/Incoming/scans/Inbox/microfilm and microfiche/microfilm` are in there? I only see those in `/Incoming/scans/Inbox/Photocopies from Hamburg`, which is a different folder. I never really understood where the images in `microfilm and microfiche` came from, I'm not sure it's from Hamburg... is it?

**Comment by karmagongde** [comment]
@eroux I am not sure of the original source of the _microfilm and microfiche images_. Nobody told me when BDRC got these images. The image quality is these are so so. I will check them against any duplication. If we do not have these texts we will catalog them.

**Comment by TBRC-Travis** [comment]
@karmagongde the microfilm material is not in the Hamburg-Sheehy backlog spreadsheet.  Please provide Work RIDs for the items in Inbox/microfilm and microfiche/microfilm

**Comment by eroux** [comment]
@karmagongde some of the images in from `microfilm` are duplicate from some of Sheehy. Here's a little summary:
- w0159142_jp2: L888/1  = [40435](https://catalogue.ngmcp.uni-hamburg.de/receive/aaingmcp_tbtdocument_00040435) = W8LS29359 in Sheehy
- w0159143_jp2: ? follow up of previous one?
- w0159144_jp2: L205/8  [47019](https://catalogue.ngmcp.uni-hamburg.de/receive/aaingmcp_tbtdocument_00047019) = W8LS29362 in Sheehy
- w0159145_jp2 + w0159146_jp2 : L291/2  [47788](https://catalogue.ngmcp.uni-hamburg.de/receive/aaingmcp_tbtdocument_00047788) *not* in Sheehy
- w0159147_jp2 = AT56/2  [68825](https://catalogue.ngmcp.uni-hamburg.de/receive/aaingmcp_tbtdocument_00068825) = W8LS29350 in Sheehy
- w0159148_jp2 : ? follow up of previous one?
- w0159149_jp2 : ?
- w01591410_jp2: L 95/1 [72250](https://catalogue.ngmcp.uni-hamburg.de/receive/aaingmcp_tbtdocument_00072250) = W8LS29356 in Sheehy

I would propose the following:
- catalog w0159145_jp2 + w0159146_jp2
- figure out what to do with the ones that don't have a reel number
- when there's a duplicate in the two folders, we can select just the best scans, we don't need to have both. I would propose though to add both sources in the source/ folder, just in case... would that be possible @TBRC-Travis ?

Thanks!

**Comment by TBRC-Travis** [comment]
as for duplicates, yes, it's fine to have duplicate copies in sources/.  usually in this kind of situation I zip up the duplicate copies and just put the zip file in sources.  then in "archive" and "images" folder we can just use the better copy of the images.

having said that, in past conversations about duplicates I thought we were leaning toward keeping them in the image set and then using BUDA's capabilities to hide them from the viewer.  of course, that doesn't make for a nice experience if people default to the tbrc.org viewer, so perhaps we're not ready to go this route yet.

this again makes the case for having some agreements about how we want to handle duplicates altogether.

**Comment by eroux** [comment]
great, thanks! 

wrt duplicates, here it's a bit different: it's the exact same microfilm that went through 2 independent digitization process, it's not duplicates in the original, microfilm or digitization process. In that case I think it makes more sense to keep the image set that looks best and discard the other entirely.

**Comment by TBRC-Travis** [comment]
oh interesting!  hmm.  a peculiar use case here.  I almost feel like we should still give the lesser "duplicate" it's on work number to reference and then find a way of restricting it from view (and noting that it's superseded by the better copy).  would there be merit in such an approach?  Otherwise I'm happy to zip up the dupes and put them in sources as mentioned before.

**Comment by eroux** [comment]
I agree it would make sense yes! I'm not really sure there's a lot to gain in our case though...

**Comment by karmagongde** [comment]
@eroux Thanks for finding work numbers in Sheehy spreadsheet. 
- w0159142_jp2: L888/1 = 40435 = W8LS29359 in Sheehy
- w0159143_jp2: ? follow up of previous one? **[Yes it is continuing of previous folder's page no. 0480, L892/6 = W8LS29246]**
- w0159144_jp2: L205/8 47019 = W8LS29362 in Sheehy
- w0159145_jp2 + w0159146_jp2 : L291/2 47788 not in Sheehy **[on image no._0270, L287/22 = W8LS29335]**
- w0159147_jp2 = AT56/2 68825 = W8LS29350 in Sheehy **[AT126/5 image no. 0212 = W8LS29353]**
- w0159148_jp2 : ? follow up of previous one? **[Yes, it is continuing of AT126/5 = W8LS29353]**. **[AT92/3 = W8LS29377].**
- w0159149_jp2 : ? **[continuing of previous one, AT92/3 = W8LS29377]**
- w01591410_jp2: L 95/1 72250 = W8LS29356 in Sheehy
- 
The images in the microfilm and microfiche folder, everyone has their work number.

**Comment by karmagongde** [comment]
W8LS29344 is a duplicate of the W2KG208035 images that are already synced. So, I am withdrawing W8LS29344.

**Comment by TBRC-Travis** [comment]
@eroux just an update for you - I've passed the Hirshberg NGMP pdfs along to Tim for cropping (https://github.com/buda-base/archive-ops/issues/247).  

They will take some time to process.  We should have them online in a few weeks.  If there's a particular urgency to get them published by a particular date please let me know and we can expedite them.

**Comment by eroux** [comment]
great, thanks! No particular urgency no, it's part of a long term plan for NGMPP

**Comment by TBRC-Travis** [comment]
Good good.  Will keep you posted.

**Comment by karmagongde** [comment]
@TBRC-Travis If W8LS29344 is on cropping queue. You can remove it. This is withdrawn in favor of W2KG208035.

**Comment by TBRC-Travis** [comment]
> If W8LS29344 is on cropping queue. You can remove it. This is withdrawn in favor of W2KG208035.

thanks @karmagongde.  I've removed it from our queue.  should I delete the images for W8LS29344 entirely or is this an instance where we should keep a copy of the images in the archive (zipping them up and keeping them in sources as mentioned previously).

**Comment by eroux** [comment]
I think we could keep a copy in src/ in that case, it's from another digitization process

**Comment by TBRC-Travis** [comment]
> I think we could keep a copy in src/ in that case, it's from another digitization process

I'd be in favor of just keeping these in the archive as W8LS29344/sources with no "images" folder.  the other option would be to zip up the images and put them in W2KG208035/sources as an alternate set of images.  what do you think?

also, I think it would be important to capture these distinctions in our metadata so we have a bit of context to understand the connection and context for these digital assets which appear to be a bit of unique use case.  _"withdrawn in favor of W2KG208035"_ is a good start but I wonder if we can leverage some facet of BUDA to make it more clear how these works are related (but different).  thoughts?

**Comment by eroux** [comment]
yes, the solution with the sources/ folder sounds good. We don't really have expressivity in BUDA wrt that... I think a note would be fine

**Comment by TBRC-Travis** [comment]
@eroux @karmagongde  I'm rather confused about how to proceed with the folders in  `/Incoming/scans/Inbox/microfilm and microfiche/microfilm` (w0159142_jp2, w0159143_jp2...etc) since there seems to be duplication with some of the Sheehy material and some of the folders contain images for multiple work numbers.  It's a bit of a mess.

@eroux if you have time it would be really helpful if you'd be able to create a spreadsheet (like you did previously) to make it clear how the material should be organized.  I'm happy to move things around into appropriate work folders I'm just not sure how to do that.

**Comment by eroux** [comment]
ok, I'll do that!

**Comment by TBRC-Travis** [comment]
thanks!

**Comment by karmagongde** [comment]
@TBRC-Travis You are right. The images in _/Incoming/scans/Inbox/microfilm and microfiche/microfilm_ all of them have their work numbers created in 2017. It is a duplicate of the Sheehy material. 

Anyway, here are the work numbers for the images in _microfilm and microfiche/microfilm_, and if we find images for all of them from Sheehy material mean we covered them.

- [ ] W8LS29240 = Reel No.L757/2
- [ ] W8LS29246 = Reel No.L892/6
- [ ] W8LS29249 = Reel No.L892/6
- [ ] W8LS29332 = Reel No.L142/4
- [ ] W8LS29335 = Reel No.L287/22
- [ ] W8LS29338 = Reel No.AT133/9
- [ ] W8LS29341 = Reel No.A36/3
- [ ] W8LS29347 = Reel No.L94/3
- [ ] W8LS29350 = Reel No.AT56/2
- [ ] W8LS29353 = Reel No.AT126/5
- [ ] W8LS29356 = Reel No.L95/1
- [ ] W8LS29359 = Reel No.L888/1
- [ ] W8LS29362 = Reel No.L205/8
- [ ] W8LS29365 = Reel No.L269/2
- [ ] W8LS29368 = Reel No.L300/2
- [ ] W8LS29371 = Reel No.L300/2
- [ ] W8LS29374 = Reel No.E1359/3
- [ ] W8LS29377 = Reel No.AT92/3
- [ ] W8LS29383 = Reel No.L301/2

**Comment by TBRC-Travis** [comment]
> @eroux if you have time it would be really helpful if you'd be able to create a spreadsheet (like you did previously) to make it clear how the material should be organized. I'm happy to move things around into appropriate work folders I'm just not sure how to do that.

@eroux still waiting on this.  when you have time that would be most wonderful.  thanks.

**Comment by eroux** [comment]
Sure! Here it is (largely based on what Karma did earlier):

- w0159142_jp2 (all) + w0159143_jp2 (img 1 to 0479): W8LS29359
- w0159143_jp2 (img 0480 to the end): W8LS29246
- w0159144_jp2: W8LS29362
- w0159145_jp2 (img 1 to 267): new work: @karmagongde  can you catalog it and give us the RID? It's [this one](https://catalogue.ngmcp.uni-hamburg.de/receive/aaingmcp_tbtdocument_00047788), the same as the others
- w0159145_jp2 (img 269 to the end)+ w0159146_jp2 (all) W8LS29335
- w0159147_jp2 (img 1 to 211): W8LS29350 
- w0159147_jp2 (img 212 to end) + w0159148_jp2 (1 to ???): W8LS29353
- w0159148_jp2 (img ??? to end) + w0159149_jp2 (all) : W8LS29377
- w01591410_jp2 (all): W8LS29356

**Comment by karmagongde** [comment]
@eroux @TBRC-Travis Here is RID W2KG228517 for w0159145_jp2 (img 1 to 267).

**Comment by TBRC-Travis** [comment]
Thank you both.  This should be all I need here.  I'll let you know if I encounter any problems in organizing this.

**Comment by eroux** [comment]
(sorry, the github web interface was buggy, I didn't mean to post the same thing multiple times)

**Comment by TBRC-Travis** [comment]
No problem.  I had the same issue.  Something was funky with it this afternoon.￼
